.gitmodules

```diff
@@ -854,6 +854,9 @@
 [submodule "vendor/grammars/language-webassembly"]
 	path = vendor/grammars/language-webassembly
 	url = https://github.com/Alhadis/language-webassembly
+[submodule "vendor/grammars/language-ring"]
+	path = vendor/grammars/language-ring
+	url = https://github.com/MahmoudFayed/atom-language-ring
```

Comment:
Your grammar has a few compatibility issues with the PCRE regex engine GitHub uses when interpreting grammars. In particular, it does not recognize `\h` and `{,n}` (replace with `[0-9A-Fa-f]` and `{0,n}` respectively).